Services/CO.CDP.OrganisationInformation.Persistence.Tests/DatabaseTenantRepositoryTest.cs

```diff
@@ -159,8 +160,17 @@ await action.Should().ThrowAsync<ITenantRepository.TenantRepositoryException.Ten
             .WithMessage("Tenant not found for the given URN: urn:nonexistent");
     }
 
-    private ITenantRepository TenantRepository()
+    private DatabaseTenantRepository TenantRepository()
+        => new(GetDbContext());
+
+    private OrganisationInformationContext? context = null;
+
+    private NpgsqlDataSource? npgsqlDataSource = null;
```

Comment:
Some warnings here

Reply:
Sorry can't see any warning @andymantell ? though I can see a improvement message to make it readonly, is that what you referring?

Reply:
done, remove the unused field